model/feature-flag/registry.yaml

```diff
@@ -10,7 +10,7 @@ groups:
         stability: experimental
         brief: The unique identifier of the feature flag.
         examples: ["logo-color"]
-      - id: feature_flag.provider_name
```

Comment:
I suggest leaving this open. Using an "id" means it is "blocked" only to use IDs, and we can potentially use it for names or other semantically valid values.
I would allow you to use the following examples:
["provider-1", "guid_here",  "provider-v1"]

Reply:
I'm not sure what you mean by "leaving this open"

Reply:
What I mean is to allow any value type to be used, such as a number, string, or GUID.

Reply:
ah I see what you mean. Unfortunately that isn't possible with the types available.